release-notes.md

```diff
@@ -0,0 +1,24 @@
+##Release Notes
+***
+* Date: 9 June, 2015
+* Product: Z-Manager Installer
+* Description: Initial release
+
+***
+* Date: 23 July, 2015
+* Product: Z-Manager Installer
+* Description:
+ * Added support for Spark 1.4.1
+ * Added new Zeppelin builds from latest master
+
+***
+* Date: 28 September, 2015
+* Product: Z-Manager Multitenancy
+* Description: Initial release of v.0.0.3
+
+***
+* Date: 26 October, 2015
+* Product: Z-Manager Installer
+* Description: Added new Zeppelin builds from latest master
```

Comment:
yes, as soon as 'multitenancy' has a binary releases. 
Right now we can use a [version number](https://github.com/NFLabs/z-manager/blob/master/multitenancy/server/main.go#L23)